<launch>
    <arg name="is_sim" default="$(optenv SIMULATION true)" />
    <arg name="robot1_name" default="recbot0" />
    <arg name="robot2_name" default="grizzly0" />
    <arg name="map_type" default="aerial" />
    <arg name="location" default="ypg" />
    <arg name="random_seed" default="0" />
    <arg name="threat_breakdown" default="false" />
    <arg name="threat_sampling" default="uniform" doc="[uniform, stealth]" />
    <arg name="num_targets" default="5" doc="number of targets to simulate"/>


    <include file="$(find waypoint_planner)/launch/simulate_threats.launch">
        <arg name="robot_name" value="$(arg robot1_name)" />
        <arg name="map_type" value="$(arg map_type)" />
        <arg name="location" value="$(arg location)" />
        <arg name="random_seed" value="$(arg random_seed)" />
        <arg name="threat_breakdown" value="$(arg threat_breakdown)" />
        <arg name="threat_sampling" value="$(arg threat_sampling)" />
        <arg name="num_targets" value="$(arg num_targets)" />
    </include>

    <include file="$(find waypoint_planner)/launch/simulate_threats.launch">
        <arg name="robot_name" value="$(arg robot2_name)" />
        <arg name="map_type" value="$(arg map_type)" />
        <arg name="location" value="$(arg location)" />
        <arg name="random_seed" value="$(arg random_seed)" />
        <arg name="threat_breakdown" value="$(arg threat_breakdown)" />
        <arg name="threat_sampling" value="$(arg threat_sampling)" />
        <arg name="num_targets" value="$(arg num_targets)" />
    </include>

</launch>